src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionStringBuilder.cs

```diff
@@ -132,7 +132,7 @@ private enum Keywords
         private bool _transparentNetworkIPResolution = DbConnectionStringDefaults.TransparentNetworkIPResolution;
         private string _networkLibrary = DbConnectionStringDefaults.NetworkLibrary;
 #else
-        internal const int DeprecatedKeywordsCount = 5;
+        internal const int DeprecatedKeywordsCount = 4;
```

Comment:
Do we know what's triggered this?

Reply:
Is is used at all??

Reply:
@ErikEJ It's only used in that Assert (count is added here:
https://github.com/dotnet/SqlClient/blob/b1f7eafd7bf6d3de6ae77c3d6a29b314b76d54de/src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionString.cs#L776)